ws-services/src/main/java/org/egov/waterconnection/validator/ValidateProperty.java

```diff
@@ -37,7 +41,7 @@ public Property getOrValidateProperty(WaterConnectionRequest waterConnectionRequ
 					"Water connection cannot be enriched without PropertyId");
 		}
 		Property property = propertyList.get();
-		if (StringUtils.isEmpty(property.getUsageCategory())) {
+		if (StringUtils.isEmpty(property.getUsageCategory()) && !(waterConnectionRequest.getRequestInfo().getUserInfo().getType().equalsIgnoreCase("SYSTEM"))) {
```

Comment:
What is this check for?

Reply:
Here we are handling the open public search for property. In property open search , we don't get all property details.
Like we don't get userCategory details in open search. So handling that situation over here.